tensorflow/python/framework/ops.py

```diff
@@ -3430,8 +3430,9 @@ def _create_op_from_tf_operation(self, c_op, compute_device=True):
     # the name will still appear in _names_in_use even though the name hasn't
     # been used. This is ok, just leave _names_in_use as-is in this case.
     # TODO(skyewm): make the C API guarantee no name conflicts.
-    if ret.name not in self._names_in_use:
-      self._names_in_use[ret.name] = 1
+    name = ret.name.lower()
```

Comment:
nit: maybe call this name_key to be consistent with unique_name and make it clear that we're not changing the actual name.